packages/plugin-axe/src/lib/runner/run-axe.ts

```diff
@@ -1,4 +1,6 @@
 import AxeBuilder from '@axe-core/playwright';
+import { createRequire } from 'node:module';
+import { dirname, join } from 'node:path';
```

Comment:
We use namespace imports `import path from 'node:path'`. I'm surprised this wasn't picked up by ESLint 😕